.github/workflows/ci.yml

```
@@ -31,10 +31,13 @@ jobs:
           node .github/actions/ci/packmode.js ${{ matrix.packmode }} > mode.json
           cat mode.json
 
-      - run: bash ./InstanceSyncSetup.sh
-        working-directory: ./automation
-      - run: pwsh ./update-server.ps1 ${{ github.ref_name }} world backups $true $false
-        working-directory: ./automation
+      - name: cd automation && bash ./InstanceSyncSetup.sh
+        run: java -jar InstanceSync.jar
+      - name: cd automation && pwsh ./update-server.ps1
+        run: |
+          cp ./automation/settings.cfg ./settings.cfg
+          cp ./automation/start-automated-server.sh ./start-automated-server.sh
+          pwsh ./automation/remove-client-mods.ps1

```

if you really insist on renaming them i would suggest accepting this one: (edit access for maintainers is on)
```suggestion
      - name: Install server
        run: |
          java -jar InstanceSync.jar
          cp ./automation/settings.cfg ./settings.cfg
          cp ./automation/start-automated-server.sh ./start-automated-server.sh
          pwsh ./automation/remove-client-mods.ps1
```